src/OrchardCore.Modules/OrchardCore.Html/Views/Media-HtmlBody.Modal.cshtml

```diff
@@ -12,7 +12,7 @@
             <div class="modal-body">
             </div>
             <div class="modal-footer">
-                <button type="button" class="btn btn-primary" id="mediaSelectButton">@T["OK"]</button>
+                <button type="button" class="btn btn-primary" id="mediaBodySelectButton">@T["OK"]</button>
```

Comment:
Because below i saw `mediaMarkdownSelectButton`, maybe here `mediaHtmlSelectButton`.
But not sure and it's only a little detail.